### DISCUSSION TITLE
How to stop auto column name uppercase

### DISCUSSION THREAD

**Discussion by AleramoAl** [dbeaver/dbeaver#21874] [Q&A] [answered]
Hi to everybody,
yesterday i tried to add a new column to a table using DBeaver using a camel case name, but every time i saved it DBeaver automatically changed the name to upper case. I have already added the column to my table using a script, but i am still curious to know if it is possible to turn off this feature on DBeaver.
Could somebody help me to solve this problem? I found some informations on internet but nothing worked.

Best regards.

**Comment by LonwoLonwo** [comment, marked as answer]
Hello @AleramoAl 

This is the default behavior, yes.

You can change it by turning on this setting:

![2023-11-15 18_53_50-DBeaver Ultimate 23 2 5 - my_schema](https://github.com/dbeaver/dbeaver/assets/45152336/66752d7f-ca5c-4251-8fda-cfb3fcf031d9)

**Comment by AleramoAl** [comment]
Thank you very much! Yesterday i spent a long time to search for an answer.

**Reply by LonwoLonwo** [reply]
Thanks for the update
